src/app/core/navbar/navbar.component.html

```diff
@@ -54,3 +59,39 @@
   </div>
 </nav>
 
+<!-- Changelog modal -->
+<div class="modal fade" id="changelog" tabindex="-1" role="dialog" aria-labelledby="changelogLabel" aria-hidden="true">
+  <div class="modal-dialog modal-dialog-centered modal-lg">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title" id="changelogLabel">Changelog</h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body" id="changelog-body">
+        <p>New features, fixes, and updates will be posted here whenever we update the website!</p>
+        <p><small>Note: Whenever we release an update to the 'signing' page, we may, at our discretion, choose to give
+          you a chance to send <em>another</em> message to each of your friends' virtrolios! This ensures that you will
+          be able to test out any new features that we develop. Keep an eye on this changelog as we will mention here if we ever do give
```

Comment:
```suggestion
          be able to test out any new features that we develop. Keep an eye on this changelog as we will mention here if we ever do give
```
Maybe instead of "test out", use something like "make use of" or "enjoy"